core/controllers/learner_dashboard.py

```diff
@@ -117,6 +117,38 @@ def get(self):
         self.render_json(self.values)
 
 
+class LearnerDashboardIdsHandler(base.BaseHandler):
+
+    @acl_decorators.can_access_learner_dashboard
+    def get(self):
+        # Get the progress of the learner - the ids of the explorations and
+        # collections completed by the user, the activities currently being
+        # completed and the ones present in the playlist of the user.
+        activity_ids = (
+            learner_progress_services.get_ids_of_activities_in_learner_dashboard( # pylint: disable=line-too-long
```

Comment:
Let's call this domain object something like "learner_dashboard_activities" and say learner_dashboard_activities = learner_progress_services.get_learner_dashboard_activities() or similar. "activity_ids" seems like a list of activity IDs and I believe we use it in other contexts to mean that, so the current formulation is slightly confusing.